### PR TITLE
ADJUST1-470-DR-Review-PADAs-page

### PR DESCRIPTION
**Pull request by pavilionsahota**

### CODE REVIEW

**Comment by ldlharper** [comment]
So the checkboxes are only used by PADAs. They originally were checkboxes on each row of the table, which is why they live in the adaTable.njk.

I think now they live outside the table, it would be best to completely remove them from the adaTable.njk and move everything checkbox related into `review-prospective.njk`

File: server/views/macros/adaTable.njk [hunk 1]
```diff
@@ -18,21 +19,26 @@ showTotalPerDateCharged: optional,  defaults to false, should the total be shown
...
-              <div class="govuk-checkboxes__item">
-                <input class="govuk-checkboxes__input row-checkbox" id="prospective-{{ isoDate }}" name="prospective" type="checkbox"value="{{ isoDate }}">
-                <label class="govuk-label govuk-checkboxes__label" for="prospective-{{ isoDate }}">Charge proved {{ ada.dateChargeProved | date("DD MMM YYYY") }}</label>
+      {% if params.cardStyle %}
+        <div class="govuk-summary-card govuk-!-margin-bottom-4">
+          <div class="govuk-summary-card__content equal-padding">
+            <h2 class="govuk-heading-m">Date charge proved {{ ada.dateChargeProved | date("DD MMM YYYY") }}</h2>
+      {% else %}
+        <div class="govuk-!-margin-bottom-6">
+          {% if params.checkboxes %}
```
Comment (ldlharper) [inline server/views/macros/adaTable.njk:28]: So I don't think there is any page that has checkboxes = true and cardStyle = false.

Comment (pavilionsahota) [reply server/views/macros/adaTable.njk:28]: Didn't know the checkboxes were only used by PADAs. I've refactored this

File: server/views/macros/adaTable.njk [hunk 2]
```diff
@@ -18,21 +19,26 @@ showTotalPerDateCharged: optional,  defaults to false, should the total be shown
...
-      <div class="govuk-!-margin-bottom-6">
-        {% if params.checkboxes %}
-          {% set isoDate = ada.dateChargeProved | date("YYYY-MM-DD") %}
-          <div class="govuk-form-group">
-            <div class="govuk-checkboxes govuk-checkboxes--small" data-module="govuk-checkboxes">
-              <div class="govuk-checkboxes__item">
-                <input class="govuk-checkboxes__input row-checkbox" id="prospective-{{ isoDate }}" name="prospective" type="checkbox"value="{{ isoDate }}">
-                <label class="govuk-label govuk-checkboxes__label" for="prospective-{{ isoDate }}">Charge proved {{ ada.dateChargeProved | date("DD MMM YYYY") }}</label>
+      {% if params.cardStyle %}
+        <div class="govuk-summary-card govuk-!-margin-bottom-4">
```
[the remark this answers is not in the document]
Comment (pavilionsahota) [reply server/views/macros/adaTable.njk:23]: Yep, all done

**Review by ldlharper — APPROVED** [review]
LGTM

**Pull request merged**